_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title:  "The OpenSearch Release Candidate is now available on GitHub"
+authors: 
+  - andhopp
+date:   2021-06-02
+categories: update
+---
+
+Today we are proud to announce the Release Candidate for version 1.0.0 of both OpenSearch (derived from Elasticsearch 7.10.2) and OpenSearch Dashboards (derived from Kibana 7.10.2). The Release Candidate includes downloadable [artifacts](https://opensearch.org/downloads.html) (Linux tars and Docker images) as well as a number of OpenSearch plugins (listed below). 
```

Comment:
proud -> excited
In first sentence: Release Candidate -> Release Candidate (RC1)